fleet/view/js/list_fleet.js

```diff
@@ -0,0 +1,792 @@
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var dir = split_url[2];
+var url_host = window.location.origin + "/" + split_url[1] + "/" + split_url[2];
+
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
+
+console.log(split_host[2]);
+var index = window.location.origin + "/" + split_url[1]
+console.log(split_url[1])
+
+//var datatable_list;
+var list_do = [];
+
+let datatable_list; //variable datatable
+
+let data_list_tbl=[], data_list_def=[], flg_data_ready = 0;
+
+let fav_id = 'tbl_fleet';
+
+let menu = [
+    [1, 'Plate Number', 'plate', 0],
+    [2, 'Truck Type', 't_type', 0],
+    [3, 'Type Of Body' , 't_cat', 0],
+    [4, 'Right Driver', 'r_dri', 0],
+    [5, 'Left Driver',	'l_dri', 0],
+    [6, 'Unit Year', 'u_year', 0],
+    [7, 'Status', 'stat', 0],
+    [8, 'Last Odometer', 'l_odo', 0],
+    [9, 'Min Temp', 'min_t', 0],
+    [10, 'Max Temp', 'max_t', 0],
+    [11, 'Fleet Age', 'fleet_ag', 0],
+    [12, 'Karoseri', 'karos', 0],
+    [13, 'Thermo Type', 'therm_type', 0],
+    [14, 'Pool Area', 'pool_ar', 0],
+    [15, 'Last Hour Refrigerator', 'lh_ref', 0]
+];
+
+let checkFilter = {};
+let listFav = [], stsDataAll = false;
+
+for (var i = 0; i < menu.length; i++) {
+    checkFilter[menu[i][2]] = [];
+}
+
+var f_def = JSON.stringify(checkFilter);
+
+var defFavorite = '<li class="dropdown-divider"></li>' +
+                '<li class="dropdown-submenu">' +
+                '' +
+                '<a  tabindex="-1" href="#" class="favorite test fav-search"><i class="fa fa-caret-up" id="caret"></i> &nbsp Save Current Search</a>' +
+                '</li>' +
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
                "</li>" +
```